src/content/docs/en/guides/migrate-to-astro/from-gitbook.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Migrating from GitBook
+description: Tips for migrating an existing GitBook project to Astro
+type: migration
+stub: true
+framework: GitBook
+i18nReady: false
+---
+
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+[GitBook](https://gitbook.com) is a web-based platform for creating and publishing documentation and books in a collaborative manner, with version control integration and customizable features.
+
+
+## Key Similarities between GitBook and Astro
+
+GitBook and Astro share some similarities that will help you migrate your project:
+
+- Both Astro and GitBook support [Markdown](/en/guides/markdown-content/). You can migrate all your existing documentation utilizing GitBook's gitsync feature.
```

Comment:
For consistency, I would change "gitsync" to "Git Sync" so it is actually written the same way as in GitBook's docs.
```suggestion
- Both Astro and GitBook support [Markdown](/en/guides/markdown-content/). You can migrate all your existing documentation utilizing GitBook's Git Sync feature.
```